pkg/loadbalancer/stream/target.go

```diff
@@ -76,19 +79,20 @@ func (t *target) Configure(identifierOffset int) error {
 	if t.fwMarks == nil {
 		t.fwMarks = []networking.FWMarkRoute{}
 	}
+	offsetId := t.identifier + identifierOffset
 	for _, ip := range t.GetIps() {
 		var fwMark networking.FWMarkRoute
-		offsetId := t.identifier + identifierOffset
 		fwMark, err := t.netUtils.NewFWMarkRoute(ip, offsetId, offsetId)
 		if err != nil {
 			return err
 		}
 		t.fwMarks = append(t.fwMarks, fwMark)
 	}
+	_ = t.targetHitsMetrics.Register(offsetId, t.nspTarget)
 	return nil
 }
 
-func (t *target) Delete() error {
+func (t *target) Delete(identifierOffset int) error {
```

Comment:
Just a simple remark: exposing the identifier offset through these methods instead of having it as a(n optional)  parameter for Newtarget() looks a bit weird on the first glance.